<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Server IP:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="serverIPLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="serverPortSpinBox">
       <property name="minimum">
        <number>22</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>8080</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>119</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectBtn">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectBtn</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidget</receiver>
   <slot>connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connect()</slot>
 </slots>
</ui>
